### DISCUSSION TITLE
CSS style not loading once site deployed

### DISCUSSION THREAD

**Discussion by othrif** [community/community#23643] [Other Feature Feedback, Questions, & Ideas]
<p>Hi there,</p>
<p>I have a problem with my CSS not loading after I deploy my site.<br>
Website: <a href="https://othrif.github.io/" rel="noopener nofollow ugc">https://othrif.github.io/</a><br>
Github repo: <a href="https://github.com/othrif/notes_othmanerifki" rel="noopener nofollow ugc">https://github.com/othrif/notes_othmanerifki</a></p>
<p>When I run a local server with hugo, the site works fine. But when I deploy it to Github pages, I loose the CSS.</p>
<p>At least for the index page, I manually changed these lines after building the site:othrif.github.io/blob/master/index.html#L41-L42</p>
<p>For some reason, I get <code>https://othrif.github.iocss</code> instead of <code>https://othrif.github.io/css</code>. this fixed the style on the index page but the rest of the pages have problems.</p>
<p>Any pointers are appreciated.</p>

**Comment by FormerlyChucks** [comment, marked as answer]
<p>Some CSS seems to be loading for me. I’ll try running it myself later, but I see Python files in there, which GitHub pages won’t run. Maybe that’s the issue?</p>
<p>Edit: I think I know what the problem is. Look at lines 41 and 42. No need to use the GitHub pages link, instead, link to it like this:</p>
<pre><code>&lt;link href="/path/to/file.css" rel="stylesheet"&gt;</code></pre>

**Comment by othrif** [comment]
<p>Thanks <a class="mention" href="/mr-steal-your-script">@Mr-Steal-Your-Script</a>, indeed that was the issue. It is working fine now!</p>
